queue_services/business-bn/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "business-bn"
+version = "0.1.0"
+description = ""
+authors = ["Patrick Wang <peinan.wang@gov.bc.ca>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.13,<4"
+flask = { version = ">=3.1.0,<4.0.0", extras = ["async"] }
+jinja2 = ">=3.1.6,<4.0.0"
+markupsafe = ">=3.0.2,<4.0.0"
+werkzeug = ">=3.1.3,<4.0.0"
+asyncio-nats-client = ">=0.11.5,<0.12.0"
+asyncio-nats-streaming = "==0.4.0"
+attrs = ">=25.3.0,<26.0.0"
+blinker = ">=1.9.0,<2.0.0"
+certifi = ">=2025.1.31,<2026.0.0"
+click = ">=8.1.8,<9.0.0"
+dpath = ">=2.2.0,<3.0.0"
+itsdangerous = ">=2.2.0,<3.0.0"
+nest-asyncio = ">=1.6.0,<2.0.0"
+protobuf = ">=3.15,<3.20"
+python-dotenv = ">=1.1.0,<2.0.0"
+psycopg2-binary = "2.9.10"
+sentry-sdk = ">=2.25.0,<3.0.0"
+six = ">=1.17.0,<2.0.0"
+urllib3 = ">=2.3.0,<3.0.0"
+launchdarkly-server-sdk = "^9.10.0"
+
+# Git-based dependencies
+business-registry-account = { git = "https://github.com/bcgov/lear.git", branch = "main", subdirectory = "python/common/business-registry-account" }
+business-model = { git = "https://github.com/bcgov/lear.git", branch = "main", subdirectory = "python/common/business-registry-model" }
+business-registry-common = { git = "https://github.com/bcgov/lear.git", branch = "main", subdirectory = "python/common/business-registry-common" }
+gcp-queue = { git = "https://github.com/bcgov/sbc-connect-common.git", subdirectory = "python/gcp-queue", branch = "main" }
+registry_schemas = { git = "https://github.com/bcgov/business-schemas.git", tag = "2.18.39" }
+sql-versioning = { git = "https://github.com/bcgov/lear.git", branch = "main", subdirectory = "python/common/sql-versioning-alt" }
+structured-logging = { git = "https://github.com/bcgov/sbc-connect-common.git", subdirectory = "python/structured-logging", branch = "main" }
+
+[tool.poetry.group.test.dependencies]
+pytest = "^8.3.5"
+pytest-cov = "6.1.0"
+pytest-asyncio = "0.26.0"
+pytest-mock = "^3.14.0"
+
```

Comment:
This is missing `[tool.poetry.group.dev.dependencies]` with ruff.
You can add this by running `poetry add --group dev ruff`

The config below is missing quite a few things. You can follow an example from one of our other PRs